newIDE/app/src/EventsSheet/SearchPanel.js

```diff
@@ -33,6 +33,7 @@ type State = {|
   searchInConditions: boolean,
   searchInEventStrings: boolean,
   searchInSelection: boolean,
+  searchCount: boolean,
```

Comment:
The name searchCount sounds like a "number of search"/"how many time a search was made".
Can you think of another name representing what this thing is. It's a boolean, so reading the name must immediately gives an idea of what true and false mean :)

Reply:
maybe isSearchCountOne or checkSearchCount? This state is just check if it is searched once or not. So isSearchCountOne seems better option.